drivers/i2c/i2c_mchp_xec.c

```diff
@@ -652,9 +944,13 @@ static int i2c_xec_init(const struct device *dev)
 		.port_sel = DT_INST_PROP(n, port_sel),			\
 		.girq_id = DT_INST_PROP(n, girq),			\
 		.girq_bit = DT_INST_PROP(n, girq_bit),			\
+		.sda_pos = DT_INST_GPIO_PIN(n, sda_gpios),		\
+		.scl_pos = DT_INST_GPIO_PIN(n, scl_gpios),		\
+		.sda_gpio_label = DT_INST_GPIO_LABEL(n, sda_gpios),	\
+		.scl_gpio_label = DT_INST_GPIO_LABEL(n, scl_gpios),	\
 		.irq_config_func = i2c_xec_irq_config_func_##n,		\
 	};								\
-	DEVICE_DT_INST_DEFINE(n, &i2c_xec_init, NULL,			\
+	DEVICE_DT_INST_DEFINE(n, &i2c_xec_init, device_pm_control_nop,	\
```

Comment:
@jvasanth1 device_pm_control_nop was alread removed 

Also, doesn't this need to use new macro I2C_DEVICE_DT_INST_DEFINE

Reply:
Fixed this!